indexing-service/src/main/java/io/druid/indexing/common/task/RealtimeIndexTask.java

```diff
@@ -593,4 +622,92 @@ public void run()
       );
     }
   }
+
+  private Appenderator newAppenderator(
```

Comment:
I don't mind making these static but will you please help me understand why you prefer them to be static versus instance methods?

Reply:
Well, I don't have a strong opinion here. It's better for me to read the code because I can expect that any member variables or methods of the class are not used inside in static methods. It's up to you.